ci/terraform/account-management/remove-account.tf

```diff
@@ -19,7 +19,7 @@ module "delete_account" {
   security_group_id         = aws_vpc.account_management_vpc.default_security_group_id
   subnet_id                 = aws_subnet.account_management_subnets.*.id
   environment               = var.environment
```

Comment:
You will need to add `EMAIL_QUEUE_URL      = aws_sqs_queue.email_queue.id` to the handler_environment_variables